src/containers/Elements/Form.js

```diff
@@ -119,7 +135,9 @@ export default compose(
   connect(makeMapStateToProps),
   autoInitialisedForm,
   reduxForm({
-    destroyOnUnmount: true,
+    destroyOnUnmount: false, // need this false to make it validate across wizard
```

Comment:
We should probably also give the form a name in this case so redux can distinguish between two forms that have similar field names?

Reply:
NodeForm passes in `form` as a prop which becomes the name of the redux-form. So as of now we have three forms that show up in redux: "setup", "Symbol(EDIT_NODE)", and "Symbol(ADD_NODE)". I can more explicitly require `form`.